java/client/src/main/java/glide/api/commands/GenericClusterCommands.java

```diff
@@ -46,13 +47,16 @@ public interface GenericClusterCommands {
      *     response (such as <em>XREAD</em>), or that change the client's behavior (such as entering
      *     <em>pub</em>/<em>sub</em> mode on <em>RESP2</em> connections) shouldn't be called using
      *     this function.
-     * @example Returns a list of all <em>pub</em>/<em>sub</em> clients:
-     *     <p><code>
-     * Object result = client.customCommand(new String[]{ "CLIENT", "LIST", "TYPE", "PUBSUB" }, RANDOM).get();
-     * </code>
      * @param args Arguments for the custom command including the command name
      * @param route Routing configuration for the command
      * @return Response from Redis containing an <code>Object</code>.
+     * @example
+     *     <pre>
+     * ClusterValue&lt;Object&gt; result = clusterClient.customCommand(new String[]{ "CONFIG", "GET", "maxmemory"}, ALL_NODES).get();
```

Comment:
Fix the &lt / &gt

Reply:
Not a typo, but there is a workaround to avoid that
javadoc follows HTML formatting rules

Reply:
you can use `{@literal }` instead
https://docs.oracle.com/javase/1.5.0/docs/tooldocs/windows/javadoc.html#%7B@literal%7D